src/main/java/uk/ac/nactem/tools/sentencesplitter/EnglishSentenceSplitter.java

```diff
@@ -256,10 +256,18 @@ public class EnglishSentenceSplitter {
 		LOWERCASETERMS.add("x");
 	}
 
+  public void addAbbreviation(String abbrev) {
```

Comment:
i'd make these `static` since they modify `static` variables. i'd also use the same formatting as is used in the file (tabs)